### PR TITLE
[Check-in] View Mixed Language disclaimer on non-english pages

### PR DESCRIPTION
**Pull request by brdunfield**
## Description
This PR:
- Adds the multiple language disclaimer on any pages loaded with a language that is not english (behind a feature flag)
- Adds the Wrapper component which cleans up the page components and removes duplicate code. The wrapper component now is in charge of the multi language disclaimer, the language switcher, and the h1 of the page.
- Many of the pages had their useEffect() block for focusing removed as this is now present in the wrapper component

New feature flag:
```
  checkInExperienceTranslationDisclaimerSpanishEnabled: 'check_in_experience_translation_disclaimer_spanish_enabled',
 ```

The framework for this flag can be easily copied for tagalog support when we get there.

I apologize for the amount of files in this PR but I did some file tree changes as well which touched a lot of things

## Original issue(s)
department-of-veterans-affairs/va.gov-team#41374
department-of-veterans-affairs/va.gov-team#41229

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/check-in/components/ErrorMessage.jsx [hunk 1]
```diff
@@ -1,19 +1,14 @@
-import React, { useEffect } from 'react';
-import { focusElement } from 'platform/utilities/ui';
+import React from 'react';
 import PropTypes from 'prop-types';
 import { useTranslation } from 'react-i18next';
 
-const ErrorMessage = ({ header, message, additionalDetails }) => {
+const ErrorMessage = ({ message, additionalDetails }) => {
   const { t } = useTranslation();
-  const errorHeader = header ?? t('we-couldnt-check-you-in');
```
Comment (brdunfield) [inline src/applications/check-in/components/ErrorMessage.jsx:8]: To make this work with the wrapper component, I had to lift the header up into the Error component.

Is there a reason for keeping these two components separate? The only place ErrorMessage seems to be used independently is ErrorBoundary and I can't tell if that's actually in use. (Also a confirmation as to whether this change will break that or not if it *is* in use would be great!)

File: src/applications/check-in/components/layout/Footer.jsx [hunk 2]
```diff
@@ -3,9 +3,9 @@ import { useSelector } from 'react-redux';
 import { useTranslation } from 'react-i18next';
```
Comment (brdunfield) [inline src/applications/check-in/components/layout/Footer.jsx:3]: I moved this file into a new layout folder with the new Wrapper component - a lot of the file changes in this PR are just updating imports

File: src/applications/check-in/components/tests/MixedLanguageDisclaimer.unit.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
+/* eslint-disable camelcase */
```
[the remark this answers is not in the document]
Comment (brdunfield) [reply src/applications/check-in/components/tests/MixedLanguageDisclaimer.unit.spec.jsx:1]: Needed for feature flag support

**Review by brianseek — APPROVED** [review]
This all looks good to me, I double checked the ErrorBoundary and this side of the feature flag looks good. I left a comment in slack about needing the feature flag in vets-api too.

**Review by it-harrison — APPROVED** [review]

**Pull request merged**